kyo-core/shared/src/test/scala/kyoTest/KyoAppTest.scala

```diff
@@ -5,18 +5,25 @@ import scala.concurrent.duration.*
 
 class KyoAppTest extends KyoTest:
 
-    "effects" in run {
-        val app = new App:
```

Comment:
Was it intentional to create a new `App` instead of a `KyoApp`?

Reply:
No, I think it's a leftover of refactorings.